gCoffee/app/src/main/java/com/mborowiec/gcoffee/Order.java

```diff
@@ -108,21 +109,36 @@ private void makeOrder(String coffee) {
     }
 
     /**
-     * Checks whether the menu contains the ordered item. If it does, asks to confirm order.
+     * Checks whether the menu contains the ordered item.
+     * @param item  item ordered by user
+     * @return boolean value
+     */
+    public boolean checkMenu(String item, String[] coffeeList) {
+        if (item != null) item = item.toLowerCase();
+        for (int i = 0; i < coffeeList.length; i++) {
+            coffeeList[i] = coffeeList[i].toLowerCase();
+        }
+
+        return Arrays.asList(coffeeList).contains(item);
+    }
+
+    /**
+     * Handles the incoming deeplink with ordered item. Checks whether the item is on the menu.
+     * If it is, asks to confirm order, if it is not, notifies the user about it.
+     * If the input is null, it  starts the Main activity instead of proceeding to order.
```

Comment:
the?

Reply:
The sentence ends with "menu", for some reason Github moved it to the next line even though it should fit fine in one line.